documentation/docs/Contributing/Contributing_to_repository/development-pipeline.md

```diff
@@ -52,7 +52,7 @@ Before you make any changes to the repository, we recommend that you create an i
 ### Select the relevant template for your report
```

Comment:
We should encourage checking of existing issues before default creating a new issue. Also creates a pathway whereby people know how to contribute even if they don't have a thing they want to change. Could we generalise this page to include, people who want to make a change but find that change on the existing issue list?

Reply:
I also thought this, it should be more clear once I separate steps onto their own pages (the description page can then point different people in the correct direction). This was harder to do within a single markdown document (and make it easy to follow)